README.md

```diff
@@ -1118,7 +1118,7 @@ The `ServiceFabricDiagnosticPipelineFactory` is a replacement for the standard `
 | `configurationFileName` | "eventFlowConfig.json" | The name of the configuration file that contains pipeline configuration. The file is expected to be part of a (Service Fabric) service configuration package.|
 | `configurationPackageName` | "Config" | The name of the Service Fabric configuration package that contains the pipeline configuration file.|
 
-The recommended place to create the diagnostic pipeline is in the service `Main()` method:
+The recommended place to create the diagnostic pipeline is in the service `Main()` method. The following code will work all types of Service Fabric services (statefull, stateless and actor):
```

Comment:
Just curious, it seems redundant, why appending this statement?

Reply:
One of the issues was about a customer that was not 100% sure how to enable EventFlow in an actor service. This is an attempt to make it extra clear.